### PR TITLE
[ci skip] Fix confusing sentence in upgrade guide

### PR DESCRIPTION
**Pull request by vishaltelangre**

### CODE REVIEW

File: guides/source/upgrading_ruby_on_rails.md [hunk 1]
```diff
@@ -319,7 +319,7 @@ By opting-out you optimize `$LOAD_PATH` lookups (less directories to check), and
 
 #### Thread-safety
 
-In classic mode constant autoloading is not thread-safe, though Rails has locks in place for example to make web requests thread-safe when autoloading is enabled, as it is common in `development` mode.
+Constant autoloading is not thread-safe in classic mode, though Rails has locks in place for example to make web requests thread-safe when autoloading is enabled, as it is common in `development` mode.
```
Comment (vishaltelangre) [inline guides/source/upgrading_ruby_on_rails.md]: The `In classic mode constant autoloading is not thread-safe` sentence is a bit confusing.

There are two ways to clear this confusion.

1. Use comma like

```diff
- In classic mode constant autoloading is not thread-safe
+ In classic mode, constant autoloading is not thread-safe
```

2. Reword the sentence to reduce the ambiguity in it. The next sentence on line 324 is the same as this.

```diff
- In classic mode constant autoloading is not thread-safe
+ Constant autoloading is not thread-safe in classic mode
```

**Comment by fxn** [comment]
Interesting, it reads straightforward to me. It might be that I am using a Catalan/Spanish construct that maybe is not idiomatic in English.

@pixeltrix should we apply this patch?

**Comment by vipulnsward** [comment]
@fxn yes, the sentence is confusing because its free running. 
I would have kept as is, but since "classic mode" implies classic vs zietwerk, this is "subject", which can use a pause instead of being free running.

I would go with:

```
+ In classic mode, constant autoloading is not thread-safe
```

**Comment by vishaltelangre** [comment]
@vipulnsward The sentence on the next line (line number 324) is like this.

```
Constant autoloading is thread-safe in `zeitwerk` mode.
``` 

Should we change it in the form you suggested above, too?

**Comment by fxn** [comment]
> Should we change it in the form you suggested above, too?

Not necessary. Indeed, one thing the current wording does is avoid repeating the same sentence structure wrt the opening one in the next paragraph.

**Comment by fxn** [comment]
@vipulnsward good, I see you guys need to have a clearly distinguished subject.

Let's go with the comma.

**Comment by vishaltelangre** [comment]
@fxn @vipulnsward Thanks for the feedback. I have updated the PR to use the sentence version with comma.

**Comment by fxn** [comment]
Perfect, thank you!

**Pull request merged**